doc/examples/shared_object_workload.json

```diff
@@ -0,0 +1,11 @@
+{
+	"tasks" : {
+		"thread0" : {
+			"loop" : 3,
+			"external_workload" : {
+                "library_name": "fpBench/fp.so",
+                "main_method": "main"
+            }
+		}
+	}
+}
```

Comment:
This doesn't seem to produce any sort of logs or numbers.
How is this supposed to be used? How the user get to know what happended or how the external workload performed?

Reply:
Made the example a bit more complex so as to show a potential user how to manage the functionality. The external_workload should behave as a regular event. The execution of the external workload produces debug output (when that log level is selected) in the rt-app.c

Reply:
This should probably be moved to doc/examples/tutorial/example9.json

Also please include a `global` section like the following
```
       "global" : {
                "calibration" : "CPU0",
                "default_policy" : "SCHED_OTHER",
                "pi_enabled" : false,
                "lock_pages" : false,
                "logdir" : "./",
                "log_basename" : "rt-app9",
                "ftrace" : false,
                "gnuplot" : false,
        }
```
and indent `tasks` as for the other examples